recipes/tibanna/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "tibanna" %}
+{% set version = "0.10.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "aa63462bb2e114442baca9cb1ee25980c3f03356d2c280ed7eabf694c79061db"
+
+build:
+  number: 0
+  entry_points:
+    - tibanna = tibanna.__main__:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: true  # [not py36]
```

Comment:
I think this recipe can be ``noarch`` and you can restrict python version on ``host`` and also on ``run`` requirements

Reply:
```suggestion
  noarch: python
```